src/pages/me/me.vue

```diff
@@ -99,9 +103,9 @@ function goLogin() {
         <div class="text">
           我的订单
         </div>
-        <div class="more">
+        <div class="more" @click="Jump('/pages/me/form')">
           <span>全部订单</span>
-          <span class="num">{{ 2 }}</span>
+          <span class="num">{{ orderNum }}</span>
```

Comment:
<span class="num">{{ orderNum }}</span>
```suggestion
<span class="num" v-if="orders">{{ orders.length }}</span>
```